src/libs/actions/Report.ts

```diff
@@ -1570,7 +1572,7 @@ function getOlderActions(reportID: string | undefined, reportActionID: string |
         reportActionID,
     };
 
-    API.paginate(
+    return API.paginate(
```

Comment:
I think this violates our philosophy, is it absolute necessary


https://github.com/Expensify/App/blob/085af64559aacfab8c87aa7bfe720fb5953b1ee4/contributingGuides/philosophies/DATA-BINDING.md?plain=1#L23